strings/accents/accent_skaven.json

```diff
@@ -0,0 +1,294 @@
+{
+
+    "Skaven": {
+
+
+
+		
+		"captain":"captain-thing",
+		"head":"head-thing",
+		"personnel":"person-things",
```

Comment:
I'd trade person for "Slaves"

Reply:
Not sure if people would like slavery being referenced in their space manz.
Willing to take other opinions

Reply:
quiet code-slave